datasets/financial_phrasebank/README.md

```diff
@@ -0,0 +1,194 @@
+consider the sentences from the view point of an investor only; i.e. whether
+the news may have positive, negative or neutral influence on the stock price.
+As a result, sentences which have a sentiment that is not relevant from an
+economic or financial perspective are considered neutral.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+The corpus used in this paper is made out of English news on all listed
+companies in OMX Helsinki. The news has been downloaded from the LexisNexis
+database using an automated web scraper. Out of this news database, a random
+subset of 10,000 articles was selected to obtain good coverage across small and
+large companies, companies in different industries, as well as different news
+sources. Following the approach taken by Maks and Vossen (2010), we excluded
+all sentences which did not contain any of the lexicon entities. This reduced
+the overall sample to 53,400 sentences, where each has at least one or more
+recognized lexicon entity. The sentences were then classified according to the
+types of entity sequences detected. Finally, a random sample of ∼5000 sentences
+was chosen to represent the overall news database.
+
+#### Who are the source language producers?
+
+The source data was written by various financial journalists.
+
+### Annotations
+
+#### Annotation process
+
+This release of the financial phrase bank covers a collection of 4840
+sentences. The selected collection of phrases was annotated by 16 people with
+adequate background knowledge on financial markets.
+
+Given the large number of overlapping annotations (5 to 8 annotations per
+sentence), there are several ways to define a majority vote based gold
+standard. To provide an objective comparison, we have formed 4 alternative
+reference datasets based on the strength of majority agreement: 
+
+#### Who are the annotators?
+
+Three of the annotators were researchers and the remaining 13 annotators were
+master's students at Aalto University School of Business with majors primarily
+in finance, accounting, and economics.
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+All annotators were from the same institution and so interannotator agreement
+should be understood with this taken into account.
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+[More Information Needed]
+
+### Licensing Information
+
+License: Creative Commons Attribution 4.0 International License (CC-BY)
+
+### Citation Information
+
+@article{Malo2014GoodDO,
+  title={Good debt or bad debt: Detecting semantic orientations in economic texts},
+  author={P. Malo and A. Sinha and P. Korhonen and J. Wallenius and P. Takala},
+  journal={Journal of the Association for Information Science and Technology},
+  year={2014},
+  volume={65}
+}
```

Comment:
````suggestion
```
@article{Malo2014GoodDO,
  title={Good debt or bad debt: Detecting semantic orientations in economic texts},
  author={P. Malo and A. Sinha and P. Korhonen and J. Wallenius and P. Takala},
  journal={Journal of the Association for Information Science and Technology},
  year={2014},
  volume={65}
}
```
````
use a code block for the citation